script/Arcueid/Day 8/08_00_ARC08_4.txt

```diff
@@ -21,18 +22,21 @@ Since he wasn't here this morning, he must have simply come to school to eat lun
 [sha:910528a1a666a6663f449ab9f022423ca0c5be21]{
 -- Page 232, Offset 20248.
 -- 社長出勤も甚だしい。
-// ???
-Even the presence of the company president feels extreme.
+// 社長出勤 implication seems to be that you're so high ranking you just show
+// up when you feel like it and expect people to just wait for you
+// https://www.weblio.jp/content/%E7%A4%BE%E9%95%B7%E5%87%BA%E5%8B%A4
+This guy only turns up if and when it suits him. Unbelievable.
 }
 [sha:d47c203e2fab742db90b67d958fbd72e74b89b6f]{
 -- Page 232, Offset 20249.
 -- もしアルクェイドの事が学校中に知れ渡っていたら、間違いなくこいつと一悶着あっただろう。
-If word of Arcueid had spread throughout the entire school, there would definitely have been a dispute between us.
+If word of Arcueid had spread throughout the entire school,
+ it definitely would have spelled trouble between the two of us.
```

Comment:
for the two of us

Reply:
I think `for the two of us` would be if there would be trouble for both he and arc, but my reading here is that Arihiko knowing Shiki is associated with Arc would cause trouble between Arihiko and Shiki.

Reply:
Got it. I misinterpreted the sentence then.

Reply:
I mean maybe that's a good indicator it should be changed? Does this read more clearly as Ari:Shiki?
> it would definitely have driven a wedge between the two of us

Reply:
Yeah, I like that better.